src/test/java/com/wikia/webdriver/common/remote/operations/SiteId.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.common.remote.operations;
+
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.remote.operations.http.NoAuthOperation;
+import lombok.Getter;
+import org.json.JSONObject;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SiteId {
+
+  public SiteId(String wikiUrl) {
+    this.wikiUrl = wikiUrl;
+    this.extractSiteIdFromSpecialVersion();
+  }
+
+  @Getter
+  private String siteId;
+  private String wikiUrl;
+
+  private void extractSiteIdFromSpecialVersion() {
+    NoAuthOperation request = new NoAuthOperation();
+    String response = request.execute(this.wikiUrl + URLsContent.SPECIAL_VERSION, new JSONObject());
+    Pattern p = Pattern.compile(".*city_id: (\\d+).*", Pattern.DOTALL);
+    Matcher m = p.matcher(response);
+    if (m.find()) {
+      this.siteId = m.group(1);
+    }
+    else {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)